entities/UserOrganization.php

```diff
@@ -41,7 +42,11 @@ public function create(){
 			$uid=$_SESSION['id'];
 		}
 
-		if( $stmt->execute(['user_id'=>$this->user_id,'organization_id'=>$this->organization_id,'admin'=>$this->admin,'user_approver'=>$this->user_approver,'need_approver'=>$this->need_approver,'manage_offers'=>$this->manage_offers,'manage_clients'=>$this->manage_clients,'client_share_approver'=>$this->client_share_approver,'confirmed'=>$this->confirmed,'confirmation_string'=>$this->confirmation_string,'user_id2'=>$uid])){
+		if($this->admin=='N'){
```

Comment:
I think this check is saying "if the user you are editing is not an admin, set dbs_check to U". It is looking at the user being edited, not the logged in user. The check should be if(!is_org_admin()|| $organization_id!=$_SESSION['organization_id'])

Reply:
I have changed the check to what you suggested, and also added an if(!is_admin) check to this check and another check we implemented to ensure that system admins can edit dbs check even if they aren't an org admin for that org.